jetty-maven-plugin/src/it/jetty-deploy-war-mojo-it/pom.xml

```diff
@@ -44,14 +45,34 @@
       </plugin>
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-surefire-plugin</artifactId>
+        <artifactId>maven-failsafe-plugin</artifactId>
```

Comment:
should this be failsafeVersion rather than surefireVersion?

Reply:
well surefire and failsafe are parts of the same source at the Maven project and they are always released together. If you prefer having an other property I don't mind just let me know.

Reply:
Not worth delaying the merge over. Merge it!